airbyte-integrations/connectors/source-braintree/source_braintree/run.py

```diff
@@ -3,12 +3,41 @@
 #
 
 
+import json
 import sys
 
 from airbyte_cdk.entrypoint import launch
 from source_braintree import SourceBraintree
 
 
 def run():
-    source = SourceBraintree()
-    launch(source, sys.argv[1:])
+    args = sys.argv[1:]
+    catalog_path = None
+    config_path = None
+    state_path = None
+
+    for i in range(len(args) - 1):
+        if args[i] == "--catalog":
+            catalog_path = args[i + 1]
+        elif args[i] == "--config":
+            config_path = args[i + 1]
+        elif args[i] == "--state":
+            state_path = args[i + 1]
+
+    catalog = None
+    if catalog_path:
+        with open(catalog_path, "r") as f:
+            catalog = json.loads(f.read())
+
+    config = None
+    if config_path:
+        with open(config_path, "r") as f:
+            config = json.loads(f.read())
+
+    state = None
+    if state_path:
+        with open(state_path, "r") as f:
+            state = json.loads(f.read())
+
+    source = SourceBraintree(catalog=catalog, config=config, state=state)
+    launch(source, args)
```

Comment:
I think this section change is what causes a lot of acceptance tests to fail. Can you look into other source connectors that run on Airbyte CDK 6.0 and that are using CDK directly (not manifest-only) and see how they implement `run` and the source class?